tests/mason-lspconfig/api/api_spec.lua

```diff
@@ -0,0 +1,55 @@
+local mason_lspconfig = require "mason-lspconfig"
+local _ = require "mason-core.functional"
+
+describe("mason-lspconfig API", function()
+    it("should return all available servers", function()
+        local server_mappings = require "mason-lspconfig.mappings.server"
+        local available_servers = mason_lspconfig.get_available_servers()
+        assert.equal(#vim.tbl_keys(server_mappings.package_to_lspconfig), #available_servers)
```

Comment:
I figured it's easier to not hard-code the number of servers, but feel to revert it if you don't want to hard-code the dependency itself :)